src/v2/Components/Artwork/Metadata.tsx

```diff
@@ -25,11 +27,12 @@ export class Metadata extends React.Component<MetadataProps> {
       hidePartnerName,
       hideArtistName,
       useLighterFont,
+      ...boxProps
     } = this.props
 
     return (
       <Link href={artwork.href} underlineBehavior="none">
-        <Box mt="12px" textAlign="left" className={className}>
+        <Box textAlign="left" className={className} {...(boxProps as any)}>
```

Comment:
The `any` here is due to a type conflict between `color` on `Box` and color from an `HTMLElement`. I'm not sure exactly _why_ it's happening in this case, but I'll revisit.

Reply:
Is it worth leaving 👆 as a comment in the code or its relatively obvious?